classes/Form/ConfigurationForm.php

```diff
@@ -0,0 +1,201 @@
+<?php
+/**
+ * 2007-2020 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\Module\Ps_Googleanalytics\Form;
+
+use AdminController;
+use Configuration;
+use HelperForm;
+use Ps_Googleanalytics;
+use Shop;
+use Tools;
+
+class ConfigurationForm
+{
+    private $module;
+
+    public function __construct(Ps_Googleanalytics $module)
+    {
+        $this->module = $module;
+    }
+
+    /**
+     * generate
+     *
+     * @return string
+     */
+    public function generate()
+    {
+        // Check if multistore is active
+        $is_multistore_active = Shop::isFeatureActive();
+
+        // Get default language
+        $default_lang = (int) Configuration::get('PS_LANG_DEFAULT');
+
+        $helper = new HelperForm();
+
+        // Module, token and currentIndex
+        $helper->module = $this->module;
+        $helper->name_controller = $this->module->name;
+        $helper->token = Tools::getAdminTokenLite('AdminModules');
+        $helper->currentIndex = AdminController::$currentIndex . '&configure=' . $this->module->name;
+
+        // Language
+        $helper->default_form_language = $default_lang;
+        $helper->allow_employee_form_lang = $default_lang;
+
+        // Title and toolbar
+        $helper->title = $this->module->displayName;
+        $helper->show_toolbar = true;        // false -> remove toolbar
+        $helper->toolbar_scroll = true;      // yes - > Toolbar is always visible on the top of the screen.
+        $helper->submit_action = 'submit' . $this->module->name;
+        $helper->toolbar_btn = [
+            'save' => [
+                'desc' => $this->module->l('Save'),
+                'href' => AdminController::$currentIndex . '&configure=' . $this->module->name . '&save' . $this->module->name .
```

Comment:
```suggestion
                'href' => AdminController::$currentIndex . '&configure=' . $this->module->name . '&save=' . $this->module->name .
```
I'm not sure about this, but without the `=` the URL looks strange ?

Reply:
well see !